picard/ui/options/plugins.py

```diff
@@ -229,46 +232,71 @@ def plugin_updated(self, plugin_name):
                 self.change_details()
                 break
 
+    def uninstall_plugin(self):
+            plugin = self.items[self.ui.plugins.selectedItems()[COLUMN_NAME]]
```

Comment:
Indentation of the method body is incorrect , 4 extra spaces.

Reply:
aah. due to vim.